packages/contracts/docs/core/01-how-it-works/01-core/02-permissions/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Permissions
+---
+
+## Managing and Governing Your DAO
+
+At Aragon, we believe that **DAOs are permission management systems**.
+Permissions between contracts and wallets allow a DAO to manage and govern its actions.
+
+Here, you will learn how the permissions in aragonOS work, how they can be granted and revoked from wallets and contracts, and how they are managed through the DAO.
+
+As we mentioned earlier, it is essential that only the right person or contract can execute a certain action. As a developer, you might have seen or used [modifiers such as `onlyOwner`](https://docs.openzeppelin.com/contracts/2.x/api/ownership#Ownable) in contracts. This `onlyOwner` modifier provides basic access control to your DAO: only the `owner` address is permitted to execute the function to which the modifier is attached.
+
+In aragonOS, we follow the same approach but provide more advanced functionality:
+Each `DAO` contracts includes a `PermissionManager` contract allowing to flexibly, securely, and collectively manage permissions through the DAO and, thus, govern its actions.
+This `PermissionManager`, called `ACL` in previous aragonOS contract versions, was one big reason why aragonOS never got hacked.
+The code and configuration of a DAO specifies which wallets or contracts (`who`) are allowed to call which authorized functions on a target contract (`where`).
+Identifiers, permissions, and modifiers link everything together.
```

Comment:
I would introduce the `auth` modifier here, and not below:

auth => permission ID => permission => rules, etc

Reply:
Not sure if I agree. Anyway, let's postpone this for now. Maybe create a task for it

Reply:
Done APP-1732